src/test/java/teammates/test/cases/ui/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -35,11 +35,29 @@ public void classSetup() {
     
     @Test
     public void testAll() throws Exception {
+        testModerationHint();
         testEditResponse();
         testAddResponse();
         testDeleteResponse();
     }
     
+    private void testModerationHint() throws Exception {
+        ______TS("verify moderation hint");
+        
+        submitPage = loginToInstructorEditStudentFeedbackPage("IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt",
+                "session1InIESFPTCourse");
+        
+        submitPage.verifyModerationHeaderHtml("/instructorEditStudentFeedbackHint.html");
+        
```

Comment:
The line `assertEquals("[More]", submitPage.getModerationHintButtonText());` is deleted here as `submitPage.verifyModerationHeaderHtml("/instructorEditStudentFeedbackHint.html");` already verified the text in the link.